app/main/lib/text_similarity.py

```diff
@@ -170,6 +174,8 @@ def search_text_by_model(search_params):
     if model_key.lower() == 'elasticsearch':
         conditions = get_elasticsearch_base_conditions(search_params, clause_count, threshold)
         language = search_params.get("language")
+        if language == 'None':
```

Comment:
thinking out loud (not for this PR): we should put all these possible values for language param somewhere so defined in one place